README.md

```diff
@@ -134,7 +133,8 @@ and the output won't be interlaced due to concurrency.
 
 ### TestResults file
 
-    nuget Expecto.TestResults
+<s>
```

Comment:
What's this :)?

Reply:
[A strikethrough](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/s). To indicate that the existing way of generating XML test results has changed.